### ISSUE TITLE
Loss functions

### ISSUE THREAD

**Issue by anmolsjoshi** [lengstrom/fast-style-transfer#102]
Had a few questions related to the loss functions. 

1. why is there a factor of two multiplied to each of the losses? 

2. regarding the style loss. the gram matrix is divided by the dimensions of the feature map, and then the style loss is divided by dimensions of the feature map again and batch size. Shouldn't it only be divided by the feature map once?

Really appreciate the help in advance!

**Comment by lengstrom** [comment]
1. It doesn't really matter, it's just nice.
2. This was originally a mistake, but training works fine anyway so I haven't fixed it yet. See #115 - pull requests welcome :)
